mdx_del_ins_markdown_3_0_1.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
Seems you're copy-pasting this entire file instead of modifying it to be compatible with markdown 3.0. Why is that?